.circleci/config.yml

```diff
@@ -564,6 +566,7 @@ workflows:
           type: approval
       - create-release-branch:
           e: default-executor
+          context: pdt-publish-restricted-context
```

Comment:
How does this work for scheduled jobs ?

Reply:
It shouldn't block scheduled jobs. 

This would only add restrictions to the scheduled job if we included this context line in the workflow 'cut-release' under 'create-release-branch'. That isn't something we want.